primitives/io/src/lib.rs

```diff
@@ -370,6 +373,13 @@ pub trait Misc {
 			.call_in_wasm(wasm, "Core_version", &[], &mut ext)
 			.ok()
 	}
+
+	/// Get the current system time in nanoseconds.
+	fn system_time() -> u128 {
```

Comment:
If I understood correctly, @gavofyork asked for this function to be available only behind a flag and it seems here it is available always. Not sure how strong that request was but still worth asking I guess?

Reply:
It is now disabled by default per the latest suggestion by Basti